jobset-generator/src/hydra_github_jobset_generator/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.1.0"
```

Comment:
Is it expected that the `__init__` is empty except for the version? Seems weird, but I'm not that familiar with Python packages / modules.

Reply:
No idea. :)